templates/templates/experiment_html.template

```diff
@@ -0,0 +1,6 @@
+=== Experiment Overview ===
+{%- if verbosity_level == "info" %}
```

Comment:
Is the verbosity level hierarchical at all? i can't imagine this does what you really want...

Consider if i have verbosity "debug" - i would still want these details and much more. You should likely have a "base" level and each additional level of detail would be conditional instead.

Reply:
potentially, i don't realize that this info isn't at the base level.... but i'd consider that to be a higher verbosity.

Please re-review your verbosity levels and make them consistent with python levels. If the names exist, reversing them in level of output is confusing and error prone.

Reply:
developer is the most verbose of the verbosity levels, so really developer probably makes the most sense. However, I think that info should be the default for now, so that is the only verbosity level I considered for now.

Would it be alright to leave it or get rid of any verbosity level implementation for now, until I get to the implementation of all the verbosity levels?